src/test/java/org/openstreetmap/atlas/checks/validation/areas/WaterbodyAndIslandSizeTest.java

```diff
@@ -0,0 +1,73 @@
+package org.openstreetmap.atlas.checks.validation.areas;
```

Comment:
Based on Intellij Coverage, it looks like you need some unit tests to cover the following cases:
1. Area is waterbody and surface area < this.waterbodyMinimumArea
2. Area is waterbody and surface area > this.waterbodyMaximumArea
3. Area is island and surface area > this.islandMaximumArea
4. Area is islet and surface area < this.isletMaximumArea
5. Relation member is role inner and area > this.islandMaximumArea
6. Relation member is role outer and area < this.waterbodyMinimumArea
7. Relation member is role out and area > this.waterbodyMaximumArea

Reply:
Let me know if you want to see how I found this out.